libs/deposits/src/lib/deposit-form.tsx

```diff
@@ -340,13 +341,17 @@ const FormButton = ({
 interface UseButtonProps {
   children: ReactNode;
   onClick: () => void;
+  className?: string;
```

Comment:
chore (blocking): This is a pointless className just apply directly

Reply:
Not sure what you mean? I was trying to change the style for the `requestFaucet` button:

`<UseButton onClick={requestFaucet} className="bottom-0 top-auto">`

This was to move it below the dropdown because the action shouldn't be appearing above the field that makes it appear, it should be after it.

Is there a way to add that extra style without adding the `className` property to `UseButtonProps`?

Reply:
Resolved as part of switching to ButtonLink.